src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2.py

```diff
@@ -319,15 +317,10 @@ def arbeitsl_geld_2_vor_vorrang_m_bg(  # noqa: PLR0913
     ----------
     arbeitsl_geld_2_regelbedarf_m_bg
         See :func:`arbeitsl_geld_2_regelbedarf_m_bg`.
-    kindergeld_m_bg
-        See :func:`kindergeld_m_bg`.
-    kind_unterh_erhalt_m_bg
-        See basic input variable
-        :ref:`kind_unterh_erhalt_m_bg <kind_unterh_erhalt_m_bg>`.
-    unterhaltsvors_m_bg
-        See :func:`unterhaltsvors_m_bg`.
     arbeitsl_geld_2_eink_m_bg
         See :func:`arbeitsl_geld_2_eink_m_bg`.
+    kindergeld_übertrag_m_bg
```

Comment:
```suggestion
    kindergeldübertrag_m_bg
```
One word, no abbreviation = no underscore.